pages/index.ts

```diff
@@ -66,7 +66,7 @@ const {
     typerighterTelemetryAdapter.matchDecorationClicked(match, document.URL),
   requestMatchesOnDocModified: true,
   adapter: new TyperighterChunkedAdapter(
-    "https://checker.typerighter.local.dev-gutools.co.uk"
```

Comment:
Not strictly related but added as part of this PR – it's much more likely developers are working with the CODE sandbox when developing PM-TR locally, and I'm forever correcting when making PRs!